sdks/python/apache_beam/io/avroio_test.py

```diff
@@ -472,10 +416,152 @@ def test_sink_transform_snappy(self):
         assert_that(readback, equal_to([json.dumps(r) for r in self.RECORDS]))
 
 
-class TestFastAvro(TestAvro):
-  def __init__(self, methodName='runTest'):
-    super(TestFastAvro, self).__init__(methodName)
-    self.use_fastavro = True
+
+
+@unittest.skipIf(sys.version_info[0] == 3 and
+                 os.environ.get('RUN_SKIPPED_PY3_TESTS') != '1',
+                 'Deprecating Avro in favor of FastAvro on Python 3 '
+                 'See BEAM-6522')
+class TestAvro(AvroBase,unittest.TestCase):
+
+    def __init__(self, methodName='runTest'):
+        super(TestAvro, self).__init__(methodName)
+        self.use_fastavro = False
+        self.SCHEMA =   parse('''
```

Comment:
Let's add the string version of this schema to the base class and parse it here.